release-tools/scripts/releasenotes.sh

```diff
@@ -22,8 +22,8 @@ set -e
 OLDIFS=$IFS
 REPO_ROOT=`git rev-parse --show-toplevel`
 ROOT=`dirname $(realpath $0)`;
-OD_VERSION=`python $REPO_ROOT/bin/version-info --od`
-ES_VERSION=`python $REPO_ROOT/bin/version-info --es`
+OD_VERSION=`python $REPO_ROOT/bin/version-info.sh --od`
+ES_VERSION=`python $REPO_ROOT/bin/version-info.sh --es`
```

Comment:
Why we need python for running shell file?

Reply:
missed

Reply:
fixed.